File: FunctionAndArrays/arrayBasics.java
import java.util.Arrays;
import java.util.Scanner;

public class arrayBasics{
    public static Scanner scn = new Scanner(System.in);
    public static void test1(){
        int n= scn.nextInt();
        int[] arr=new int[n];

        for(int i=0; i<arr.length;i++){
            arr[i]= scn.nextInt();
        }
        for(int i=0; i<arr.length; i++){
            System.out.print(arr[i]+" ");
        }
    
    }
    public static void display(int[] a) {
        StringBuilder sb = new StringBuilder();

        for (int val: a) {
            //for each loop
            sb.append(val + "\n");
        }
        System.out.println(sb);
    }
    public static void test2(){
        int[] arr = new int[10];
        Arrays.fill(arr,8);
        for(int i=0; i<arr.length; i++){
            System.out.print(arr[i]+" ");
        }
    }

    public static void main(String[] args){
        test1();
    }
}

File: FunctionAndArrays/arrayBasics.cpp
#include<iostream>
#include<vector>
using namespace std;

void display(vector<int>& arr){
    for(int i=0; i < arr.size(); i++){
        cout<<arr[i]<<" ";
    }
}
void test1(){
    int n; cin >> n;
    vector<int> arr(n);

    for(int i=0; i < n; i++){
        cin >> arr[i];
    }

    display(arr);
}
void test2(){
        vector <int> arr(10,8);
        display(arr);
    }
int main(int argc, char **argv){
    test2();
    return 0;
}